picoquictest/tls_api_test.c

```diff
@@ -5116,7 +5116,7 @@ int virtual_time_test()
         {
             /* Check that the simulated time follows the simulation */
             for (int i = 0; ret == 0 && i < 5; i++) {
-                simulated_time += 12345000;
+                simulated_time += 12345678000;
```

Comment:
I have a doubt.  In my mind, simulated_time was of same unity as picoquic_get_quic_time which did not change